sdk/deviceregistry/azure-resourcemanager-deviceregistry/SAMPLE.md

```diff
@@ -0,0 +1,2 @@
+# Code snippets and samples
+
```

Comment:
The sample files is generated, but this file is empty?

Reply:
It's due to sample files generated after local repo fix(not sdkIntegration). Let me do a full regen..